torchvision/models/detection/transform.py

```diff
@@ -171,14 +212,23 @@ def resize(
         target: Optional[Dict[str, Tensor]] = None,
     ) -> Tuple[Tensor, Optional[Dict[str, Tensor]]]:
         h, w = image.shape[-2:]
-        if self.training:
-            if self._skip_resize:
-                return image, target
-            size = float(self.torch_choice(self.min_size))
+        if not torch.jit.is_scripting():
+            if self.training:
+                if self._skip_resize:
+                    return image, target
+                size = random.choice(self.min_size)
```

Comment:
tests failing because this needs an import. But please use the RNG from `torch` instead of Python, so that the RNG can be controlled via `torch.manual_seed()` and similar mechanisms.